Program.cs

```diff
@@ -75,13 +111,20 @@ static void AddCharacter()
             }
         }
 
-        static void EnterName()
+        static void EnterName(Character character)
+        {
+            character.Name = Prompt("Enter the character's name");
+        }
```

Comment:
Here we are using our `Prompt()` helper to set the value of the the character's name property.